packages/components/Notification/src/NotificationTokens.ios.ts

```diff
@@ -1,7 +1,85 @@
+import { NotificationTokens } from './Notification.types';
 import { DynamicColorIOS } from 'react-native';
 
+export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme> = (t: Theme) =>
+  ({
+    backgroundColor: t.colors.background,
+    borderColor: 'transparent',
+    borderRadius: 12,
+    borderWidth: 1,
+    color: t.colors.brandForeground1,
+    fontFamily: 'primary',
+    fontLetterSpacing: -0.24, // iOS only prop
+    fontLineHeight: 20,
+    fontSize: 15,
+    fontWeight: '600',
+    padding: 16,
+    hasTitle: {
+      fontLetterSpacing: -0.08, // iOS only prop
+      fontLineHeight: 18,
+      fontSize: 13,
+      fontWeight: '400',
+      paddingVertical: 12,
+    },
+    isBar: {
+      borderRadius: 0,
+      fontWeight: '400',
+    },
+    primary: {
+      backgroundColor: notificationColors.brandBackground4,
+      color: notificationColors.brandForeground4,
+      // color: '#0F6CBD',
+      disabledColor: notificationColors.brandForegroundDisabled,
+      /**
+       * None of the foreground tokens here have pressed versions so the foreground color with an alpha value was used.
+       * The FluentUI Apple NotificationView was used to color match.
+       */
+      pressedColor: '#0F6CBD30',
+    },
+    neutral: {
+      backgroundColor: notificationColors.background4,
+      color: notificationColors.foreground2,
+      disabledColor: notificationColors.foregroundDisabled2,
+      pressedColor: '#61616145',
+    },
+    danger: {
+      backgroundColor: notificationColors.PaletteRedBackground1,
+      color: notificationColors.PaletteRedForeground1,
+      pressedColor: '#BC2F3433',
+    },
+    warning: {
+      backgroundColor: notificationColors.PaletteYellowBackground1,
+      color: notificationColors.PaletteYellowForeground1,
+      // color: '#4C4400',
+      pressedColor: '#4C440033',
+    },
+    primaryBar: {
+      backgroundColor: notificationColors.brandBackground4,
+      borderWidth: 0,
+      color: notificationColors.brandForeground4,
+      disabledColor: notificationColors.brandForegroundDisabled,
+      pressedColor: '#0F6CBD30',
+    },
+    primaryOutlineBar: {
+      backgroundColor: notificationColors.background1,
+      borderColor: notificationColors.stroke2,
+      color: notificationColors.brandForeground1,
+      disabledColor: notificationColors.brandForegroundDisabled,
+      pressedColor: '#0F6CBD30',
+    },
+    neutralBar: {
+      backgroundColor: notificationColors.background5,
+      borderWidth: 0,
+      color: notificationColors.foreground2,
+      disabledColor: notificationColors.foregroundDisabled2,
+      pressedColor: '#61616145',
+    },
+  } as NotificationTokens);
+
 /**
- * Fluent 2 colors are not yet tokenized for iOS so DynamicColorIOS is necessary for dark mode
+ * Fluent 2 colors are not yet in the token pipeline for iOS so DynamicColorIOS is necessary for dark mode
  */
 export const notificationColors = {
```

Comment:
Nit, these don't need to be exported since they're only referenced in this file. Also, we _could_ move it to the top of the file so we see they're declared before using them, but idk it matters. That's more personal preference.